flask_server.py

```diff
@@ -110,6 +84,57 @@ def logs():
 
 ################### API ROUTES ###################
 
+def get_index_settings(user_settings):
```

Comment:
These are not only the index settings, there is also the current status. Move this to two different api call. 

One for settings (loop_mode, media_mode etc) and one for the status (output, scraper_status)

Reply:
You already have status, i'd make it post and get